### PR TITLE
Add datrie to conda-forge

### PR DESCRIPTION
**Pull request by melund**
This is dependency to snakemake which I plan
make a conda-forge package for.

### CODE REVIEW

**Comment by melund** [comment]
@conda-forge-admin, please rerender

**Comment by melund** [comment]
@conda-forge-admin, please rerender

File: recipes/datrie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a11371cc2dbbad71d6dfef57ced6e8b384bb377eeb847c63d58f8dc8e8b2023
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: 
+    - cython src/datrie.pyx
```
Comment (jakirkham) [inline recipes/datrie/meta.yaml]: Why is this needed?

File: recipes/datrie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - datrie
+
+about:
+  home: https://github.com/kmike/datrie/
+  license: LGPLv2
+  license_family: LGPL
+  license_file: COPYING
+  summary: Super-fast, efficiently stored Trie for Python, uses libdatrie
+  doc_url: https://pypi.python.org/pypi/datrie/0.1.1
```
Comment (jakirkham) [inline recipes/datrie/meta.yaml]: Should we drop the version from here? Also is this what they recommend for docs?

**Comment by jakirkham** [comment]
Thanks @melund. Couple comments above. Otherwise LGTM.

File: recipes/datrie/meta.yaml (hunk 1 above)
Comment (melund) [reply recipes/datrie/meta.yaml]: Not sure. This what was done in the corresponding recipe for the windows anaconda package. I don't mind removing to see if it works without.

File: recipes/datrie/meta.yaml (hunk 2 above)
Comment (melund) [reply recipes/datrie/meta.yaml]: Yes. I will remove it.

**Comment by melund** [comment]
@jakirkham ping :)

File: recipes/datrie/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/datrie/meta.yaml]: Ah the [build script expects a C file]( https://github.com/pytries/datrie/blob/0.7.1/setup.py#L49-L54 ). Usually the build script would handle Cythonizing too, but I see we have to do it ourselves. Might be worth raising an issue upstream to ask them to handle this in `setup.py`.

File: recipes/datrie/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a11371cc2dbbad71d6dfef57ced6e8b384bb377eeb847c63d58f8dc8e8b2023
+
+build:
+  number: 0
+  skip: True  # [py27]
```
Comment (jakirkham) [inline recipes/datrie/meta.yaml]: `py27` -> `py2k`

File: recipes/datrie/meta.yaml (hunk 2 above)
Comment (jakirkham) [reply recipes/datrie/meta.yaml]: Thanks :)

**Comment by melund** [comment]
@jakirkham Done
